osu.Game/Online/API/APIAccess.cs

```diff
@@ -203,6 +207,13 @@ private void run()
                             continue;
                         }
 
+                        // populating ruleset statistics
+                        var usersRequest = new GetUsersRequest(new[] { localUser.Value.Id });
+
+                        usersRequest.Success += res => statistics.Value = res.Users.First().RulesetsStatistics;
```

Comment:
We already have a `GetUserRequest` above, does that not provide the ruleset statistics or something?

Reply:
It's only for one ruleset not all four. See the deleted part in difficulty recommender where get user was called 4 times

Reply:
In-fact it doesn't seem to contain any if you don't provide a ruleset? Neither the default ruleset or anything, seems kinda weird.

Reply:
Checking with @nanaya , it's probably simple enough to change `/me` to return all rulesets, and use that instead? Would save extra API requests and complication.